### PR TITLE
remove mermaid chart

### PR DESCRIPTION
**Pull request by b00f**
This PR removes the mermaid chart.

### CODE REVIEW

**Comment by Ja7ad** [comment]
@b00f Why remove mermaid?

File: PIPs/tx-data
Comment (Ja7ad) [inline PIPs/tx-data:1]: @b00f What is this file? for what PIPs?

Comment (b00f) [reply PIPs/tx-data:1]: It is a mistake. Let me remove it first.

**Comment by b00f** [comment]
@Ja7ad Please check again

**Review by Ja7ad — APPROVED** [review]

**Pull request merged**